### PR TITLE
C module

### PR DESCRIPTION
**Pull request by shashiranjan84**
Using libcurl library

### CODE REVIEW

**Comment by ahmadnassri** [comment]
can you add some line spacing for better readability?

**Comment by shashiranjan84** [comment]
spacing in js file or generated snippet?

**Comment by ahmadnassri** [comment]
in generated output.

**Comment by shashiranjan84** [comment]
done.

File: src/targets/c/libcurl.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+'use strict'
+
+var CodeBuilder = require('../../helpers/code-builder')
+
+module.exports = function (source, options) {
+  var code = new CodeBuilder()
+
+  code.push('CURL *hnd = curl_easy_init();')
+  code.push('')
```
Comment (ahmadnassri) [inline src/targets/c/libcurl.js:9]: there's a helper called `blank()` that does this instead.

**Pull request merged**